packages/tldraw/src/lib/shapes/note/NoteShapeUtil.tsx

```diff
@@ -47,6 +50,33 @@ export class NoteShapeUtil extends ShapeUtil<TLNoteShape> {
 	override hideResizeHandles = () => true
 	override hideSelectionBoundsFg = () => false
 
+	override canReceiveNewChildrenOfType = (shape: TLNoteShape, type: string) => {
```

Comment:
for later, feels like eventually there would be a parent class that NoteShapeUtil could extend from, like, SurfaceShapeUtil that other things could extend from and use this behavior